netplugin/plugin/netplugin.go

```diff
@@ -220,3 +196,43 @@ func (p *NetPlugin) GetEndpointStats() ([]byte, error) {
 func (p *NetPlugin) InspectState() ([]byte, error) {
 	return p.NetworkDriver.InspectState()
 }
+
+//GlobalFwdModeUpdate update the forwarding mode
+func (p *NetPlugin) GlobalFwdModeUpdate(cfg Config) {
+	var err error
+
+	if p.NetworkDriver != nil {
```

Comment:
Agreed that we have that check. But that does not necessarily guarantee everything will be ok at the netplugin. If we somehow had an endpoint on the OVS/ofnet, what would be the behavior?